.github/ISSUE_TEMPLATE/question.md

```diff
@@ -7,7 +7,7 @@ labels: kind/question
 
 <!-- Please use this template while asking a new question. Thanks!
 Please make sure you have read the FAQ and searched the issue list.
-- FAQ: https://github.com/karmada-io/karmada/blob/master/docs/frequently-asked-questions.md
+- FAQ: https://github.com/karmada-io/website/blob/main/docs/faq/faq.md
```

Comment:
```suggestion
- FAQ: https://karmada.io/docs/faq
```
How about link to the website?

Reply:
Same as below.
I would tend to link to the website because the content in a repo might change over time.

Reply:
Rebased.